openssl/src/ssl/test/mod.rs

```diff
@@ -1422,3 +1422,50 @@ fn add_chain_cert() {
     let mut ssl = Ssl::new(&ctx).unwrap();
     assert!(ssl.add_chain_cert(cert).is_ok());
 }
+#[test]
+#[cfg(any(ossl111))]
+#[cfg(not(libressl))]
+#[cfg(not(boringssl))]
```

Comment:
This should just be:
```suggestion
#[cfg(ossl111)]
```
Also can you get away with `ossl102`? I can't see what 1.1.1 functions you're using.